(venv) denham@DESKTOP-BUISKQU:~/workspace/spotify-user-matching/server$ docker logs 3cbc40e01a03
INFO:     Will watch for changes in these directories: ['/usr/src/api']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [1] using statreload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 68, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/local/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 76, in serve
    config.load()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 456, in load
    self.loaded_app = import_from_string(self.app)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 24, in import_from_string
    raise exc from None
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/api/./main.py", line 5, in <module>
    from schema import schema
  File "/usr/src/api/./schema.py", line 3, in <module>
    from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/__init__.py", line 1, in <module>
    from .types import SQLAlchemyObjectType
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/types.py", line 12, in <module>
    from .converter import (
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/converter.py", line 9, in <module>
    from .fields import createConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/fields.py", line 2, in <module>
    from promise import is_thenable, Promise
ModuleNotFoundError: No module named 'promise'
INFO:     Stopping reloader process [1]
INFO:     Will watch for changes in these directories: ['/usr/src/api']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [1] using statreload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 68, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/local/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 76, in serve
    config.load()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 456, in load
    self.loaded_app = import_from_string(self.app)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 24, in import_from_string
    raise exc from None
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/api/./main.py", line 5, in <module>
    from schema import schema
  File "/usr/src/api/./schema.py", line 3, in <module>
    from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/__init__.py", line 1, in <module>
    from .types import SQLAlchemyObjectType
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/types.py", line 12, in <module>
    from .converter import (
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/converter.py", line 9, in <module>
    from .fields import createConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/fields.py", line 2, in <module>
    from promise import is_thenable, Promise
ModuleNotFoundError: No module named 'promise'
INFO:     Stopping reloader process [1]
INFO:     Will watch for changes in these directories: ['/usr/src/api']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [1] using statreload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 68, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/local/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 76, in serve
    config.load()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 456, in load
    self.loaded_app = import_from_string(self.app)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 24, in import_from_string
    raise exc from None
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/api/./main.py", line 5, in <module>
    from schema import schema
  File "/usr/src/api/./schema.py", line 3, in <module>
    from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/__init__.py", line 1, in <module>
    from .types import SQLAlchemyObjectType
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/types.py", line 12, in <module>
    from .converter import (
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/converter.py", line 9, in <module>
    from .fields import createConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/fields.py", line 2, in <module>
    from promise import is_thenable, Promise
ModuleNotFoundError: No module named 'promise'
INFO:     Stopping reloader process [1]
INFO:     Will watch for changes in these directories: ['/usr/src/api']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [1] using statreload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 68, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/local/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 76, in serve
    config.load()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 456, in load
    self.loaded_app = import_from_string(self.app)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 24, in import_from_string
    raise exc from None
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/api/./main.py", line 5, in <module>
    from schema import schema
  File "/usr/src/api/./schema.py", line 3, in <module>
    from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/__init__.py", line 1, in <module>
    from .types import SQLAlchemyObjectType
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/types.py", line 12, in <module>
    from .converter import (
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/converter.py", line 9, in <module>
    from .fields import createConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/fields.py", line 2, in <module>
    from promise import is_thenable, Promise
ModuleNotFoundError: No module named 'promise'
INFO:     Stopping reloader process [1]
INFO:     Will watch for changes in these directories: ['/usr/src/api']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [1] using statreload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 68, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/local/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 76, in serve
    config.load()
  File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 456, in load
    self.loaded_app = import_from_string(self.app)
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 24, in import_from_string
    raise exc from None
  File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 21, in import_from_string
    module = importlib.import_module(module_str)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/api/./main.py", line 5, in <module>
    from schema import schema
  File "/usr/src/api/./schema.py", line 3, in <module>
    from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/__init__.py", line 1, in <module>
    from .types import SQLAlchemyObjectType
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/types.py", line 12, in <module>
    from .converter import (
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/converter.py", line 9, in <module>
    from .fields import createConnectionField
  File "/usr/local/lib/python3.8/site-packages/graphene_sqlalchemy/fields.py", line 2, in <module>
    from promise import is_thenable, Promise
ModuleNotFoundError: No module named 'promise'
INFO:     Stopping reloader process [1]
INFO:     Will watch for changes in these directories: ['/usr/src/api']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [1] using statreload